nuto/math/EigenOdeintCompatibility.h

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include "Eigen/Core"
 #include <boost/numeric/odeint/external/eigen/eigen_algebra_dispatcher.hpp>
```

Comment:
I think it is better to just exchange the two includes. 
1) This would resolve the problem since eigen_resize includes Eigen/Dense
2) Other prerequisites of the eigen_algebra_dispatcher are also included there, namely:
- #include <boost/core/enable_if.hpp>
- #include <boost/type_traits/is_base_of.hpp>
- #include <boost/numeric/odeint/algebra/algebra_dispatcher.hpp>
#include <boost/numeric/odeint/algebra/vector_space_algebra.hpp>